app/routes/dashboard.php

```diff
@@ -1,6 +1,13 @@
 <?php
 
-Route::group(['before' => 'auth', 'prefix' => 'dashboard', 'namespace' => 'CachetHQ\Cachet\Http\Controllers'], function () {
+/**
+ * TODO: Use Resource Controllers. This is a mess.
```

Comment:
You could use a `@todo` annotation to be really fancy, otherwise, you'll need `/*` rather than `/**`.